app/routes/report-routes.ts

```diff
@@ -95,9 +96,12 @@ const extractTestResult = (req: Request): TestResult | undefined => {
 };
 
 const createReportRateLimit = rateLimit({
-  max: 20, // allowed requests per window
+  max: config.RATE_LIMIT, // allowed requests per window
   windowMs: 24 * 60 * 60 * 1000, // 24 hour window,
```

Comment:
Maybe we should add `windowMs` to the config as well? Makes sense as `RATE_LIMIT` option by itself doesn't really indicate how strict it is.

Reply:
@michaelmcmillan perfect idea! I've added it in the latest commit, now we have:
`RATE_LIMIT_COUNT` and `RATE_LIMIT_WINDOW` new params - feel free to suggest better names :-)

I might later add config params README as the number of params starts to grow :sweat_smile: